app/controllers/edit_merchant_details/post-edit-controller.js

```diff
@@ -38,19 +39,22 @@ module.exports = (req, res) => {
     return form
   }, {})
 
-  const reqMerchantDetails = {
-    name: formFields[MERCHANT_NAME],
-    telephone_number: formFields[TELEPHONE_NUMBER].replace(/\s/g, ''),
-    email: formFields[MERCHANT_EMAIL],
-    address_line1: formFields[ADDRESS_LINE1],
-    address_line2: formFields[ADDRESS_LINE2],
-    address_city: formFields[ADDRESS_CITY],
-    address_postcode: formFields[ADDRESS_POSTCODE],
-    address_country: formFields[ADDRESS_COUNTRY]
-  }
+  formFields[TELEPHONE_NUMBER] = formFields[TELEPHONE_NUMBER].replace(/\s/g, '')
+
+  const serviceUpdateRequest = new ServiceUpdateRequest()
+    .addUpdate(validOps.replace, validPaths.merchantDetails.name, formFields[MERCHANT_NAME])
```

Comment:
As this request builder understands the context of the operations would it be appropriate to add alias methods for them (mostly just syntactic sugar)? It might read well and would mean not having to import the operations.

Example: 
```js
const serviceUpdateRequest = new ServiceUpdateRequest()
  .replace(paths.merchant.name, formFields[MERCHANT_NAME])
  .replace()
  .add()
  .formatPayload()
```